OpenSim/Simulation/SimbodyEngine/PointConstraint.cpp

```diff
@@ -150,42 +153,50 @@ void PointConstraint::setBody2PointLocation(Vec3 location)
 }
 
 /** Set the point locations */
-void PointConstraint::setContactPointForInducedAccelerations(const SimTK::State &s, Vec3 point)
+void
+PointConstraint::setContactPointForInducedAccelerations(const SimTK::State &s,
+                                                        Vec3 point)
 {
     // make sure we are at the position stage
     getSystem().realize(s, SimTK::Stage::Position);
 
     const PhysicalFrame& body1 =
-        getConnector<PhysicalFrame>("body_1").getConnectee();
+        getSocket<PhysicalFrame>("body_1").getConnectee();
     const PhysicalFrame& body2 =
-        getConnector<PhysicalFrame>("body_2").getConnectee();
+        getSocket<PhysicalFrame>("body_2").getConnectee();
 
-    // For external forces we assume point position vector is defined wrt foot (i.e., _body2)
-    // because we are passing it in from a prescribed force.
+    // For external forces we assume point position vector is defined wrt foot
+    // (i.e., _body2) because we are passing it in from a prescribed force.
     // We must also get that point position vector wrt ground (i.e., _body1)
     Vec3 spoint = body2.findLocationInAnotherFrame(s, point, body1);
 
     setBody1PointLocation(spoint);
     setBody2PointLocation(point);
 }
 
-void PointConstraint::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
+void PointConstraint::updateFromXMLNode(SimTK::Xml::Element& aNode,
+                                        int versionNumber)
```

Comment:
Do we need to increment the version number and add a new `if` block to convert from "Connector_PhysicalFrame_" to "Socket_PhysicalFrame_"? Same comment applies to other `updateFromXMLNode()` methods.